exporter/views.py

```diff
@@ -35,13 +37,81 @@ def get_original_asset_id(download_url):
     """
     if download_url.startswith("http://tile.loc.gov/"):
         pattern = r"/service:([A-Za-z0-9:\-]*)/"
-        asset_id = re.search(pattern, download_url)
+        asset_id = re.search(pattern, download_url).group(1)
```

Comment:
This will fail if it gets a value which doesn't match the pattern. We should probably do the `if not match: …` check even if that's unlikely.

Reply:
I changed it - let me know what you think.